search-sentiments-map/public/style.css

```diff
@@ -1,5 +1,11 @@
 body {
+  background-color: #c4f2e3;
+}

```

Comment:
I thought we agreed to get rid of the background color.

Reply:
Honestly I still think it looks better than white. We can decide on what color to use later.